apps/demo-react/components/WalletsModal.tsx

```diff
@@ -13,6 +13,8 @@ export default function WalletsModal(props: { isDarkTheme: boolean }) {
       shouldInvertWalletIcon={isDarkTheme}
       linkDontHaveWallet={LINK_DONT_HAVE_WALLET_DEFAULT}
       walletsPinned={['okx', 'browserExtension']}
+      onClickWalletsMore={() => console.log('metrics: wallets modal show more')}
```

Comment:
Why do we need console.log here?

Reply:
That's for demo-page demonstration purposes, same as here https://github.com/lidofinance/reef-knot/blob/851b86b093a1c3d2978820bc3fb523209ec1a5c2/apps/demo-react/util/metrics.ts#L4